flow/layers/layer.h

```diff
@@ -171,7 +176,7 @@ class Layer {
     paint_bounds_ = paint_bounds;
   }
 
-  bool needs_painting() const { return !paint_bounds_.isEmpty(); }
+  bool needs_painting() const { FML_DCHECK(false); return false; }
```

Comment:
This stub allows the unittests to compile even though they will currently fail if you run them. It will eventually be removed when the unittests are modified to test the new culling policies.

Reply:
The `FML_DCHECK` is to verify that this code is not used during application runs. I ran the test case in debug mode and it never triggered, so the only references remaining should all be in the tests.

Reply:
This stub is now gone (replaced by a similar method that takes a PaintContext).